source/includes/_fiat_to_crypto_flow.md

```diff
@@ -218,21 +218,50 @@ So the 2 potential cases are:
   - You can either not send the `kyc_verified` parameter or send it with a `null` value.
 
 
-### Status callback
+## Notification of successful transaction
 
-In order to receive status callbacks for an operation, a `callback_url` should be provided in the request. **ON/RAMP** will call back the merchant with the following JSON payload:
+A URL provided by the merchant in a previous request (see `callback_url` in [Create fiat to crypto invoice](#create-fiat-to-crypto-invoice)) will be used to send a notification request once the transaction has been confirmed.  The merchant is expected to respond with 200 status code.
+
+<aside class="success"><b><code>POST callback_url</code></b></aside>
+
+> Example cURL
+
+```shell
+
+curl -X POST callback_url \
+-H 'Content-Type: application/json' \
+-d '{
+    "invoice_id": "10e35c16-3ff8-4238-b5b5-34b4da9b4115",
+    "timestamp_utc": "2020-01-01 12:45:00.000000+00",
+    "status": {
+        "completed": {}
+    }
+}'
+```
+
+> Request JSON Body
```

Comment:
Actually, it does not reflect the actual situation. In reality, we will send a typical fiat Ingres callback not this one, it was probably wrongly copied from s2s egress flow which we were finalizing at that time also. I will update API docs on separate PR later.